### PR TITLE
fix: cursor pointer for clickables and update tags with proper tags

### PR DESCRIPTION
**Pull request by justinbalaguer**
image

this pr mostly fixes dev warnings (by using proper tags and removing unnecessary code) and added addition ux updates (mainly clickables should be pointer for cursor)

### CODE REVIEW

**Comment by lolpro11** [comment]
Does this change anything else about the frontend?

**Comment by kylerchin** [comment]
Woah! Nice PR! Can you change the alt for the north icon to say something like "Align Map North" or something similar?

File: src/routes/+layout.svelte
Comment (lolpro11) [inline src/routes/+layout.svelte:1]: Did you change app.css to have Noto Sans?

Comment (kylerchin) [reply src/routes/+layout.svelte:1]: I don't see a font change here

**Review by lolpro11 — APPROVED** [review]

**Comment by justinbalaguer** [comment]
> Does this change anything else about the frontend?

no it doesnt. only the clickable cursor as pointer

File: src/routes/+layout.svelte
Comment (justinbalaguer) [reply src/routes/+layout.svelte:1]: i didn't, you want me to include it?

Comment (lolpro11) [reply src/routes/+layout.svelte:1]: Does not having Noto Sans in [src/routes/+layout.svelte](https://github.com/CatenaryMaps/catenary-frontend/pull/16/files/b5f7d0cb3d3bce374141a685b6a09508431362ed#diff-cc69d6a97e4f62578028d872b8d5032f2786fe1eaa65735429286b375234168d) change anything?

**Review by kylerchin — APPROVED** [review]
okay looks good now

**Review by lolpro11 — APPROVED** [review]

**Comment by justinbalaguer** [comment]
let me know if there is anything to be fixed on this pr. thanks guys.

**Comment by kylerchin** [comment]
Thanks for the fix! Great job! :)

**Comment by kylerchin** [comment]
Our Discord if you want to join: https://discord.gg/6BjnxthQ

**Comment by justinbalaguer** [comment]
btw the discord link is invalid for me. @kylerchin

**Comment by lolpro11** [comment]
https://discord.gg/8cBSxdXX

**Comment by lolpro11** [comment]
> btw the discord link is invalid for me. @kylerchin
https://discord.gg/8cBSxdXX

**Comment by lolpro11** [comment]
@justinbalaguer https://discord.gg/8cBSxdXX

**Pull request merged**